test/x86/sse4.2.c

```diff
@@ -24,9 +24,231 @@
 #define SIMDE_TESTS_CURRENT_ISAX sse4_2
 #include <test/x86/test-x86-internal.h>
 #include <simde/x86/sse4.2.h>
+#include <assert.h>
 
 #if defined(SIMDE_X86_SSE4_2_NATIVE) || defined(SIMDE_NO_NATIVE) || defined(SIMDE_ALWAYS_BUILD_NATIVE_TESTS)
 
+#if defined(SIMDE_X86_SSE4_2_NATIVE)
+#define SIMDE_SIDD_UBYTE_OPS _SIDD_UBYTE_OPS
+#define SIMDE_SIDD_UWORD_OPS _SIDD_UWORD_OPS
+#define SIMDE_SIDD_SBYTE_OPS _SIDD_SBYTE_OPS
+#define SIMDE__SIDD_SWORD_OPS _SIDD_SWORD_OPS
```

Comment:
Stray underscore; this should be `SIMDE_SIDD_SWORD_OPS` not `SIMDE__SIDD_SWORD_OPS`